cpu/atxmega/include/cpu_clock.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *               2014 Freie Universität Berlin, Hinnerk van Bruinehsen
+ *               2018 RWTH Aachen, Josua Arndt <jarndt@ias.rwth-aachen.de>
+ *               2021 Gerson Fernando Budke <nandojve@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_atxmega
+ * @brief       Common implementations and headers for ATxmega family based micro-controllers
+ * @{
+ *
+ * @file
+ * @brief       Basic definitions for the ATxmega common clock module
+ *
+ * When ever you want to do something hardware related, that is accessing MCUs registers directly,
+ * just include this file. It will then make sure that the MCU specific headers are included.
+ *
+ * @author      Stefan Pfeiffer <stefan.pfeiffer@fu-berlin.de>
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Hinnerk van Bruinehsen <h.v.bruinehsen@fu-berlin.de>
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ * @author      Josua Arndt <jarndt@ias.rwth-aachen.de>
+ * @author      Gerson Fernando Budke <nandojve@gmail.com>
```

Comment:
Same here.
This is less about vanity and more about who's responsible for that family and can be bothered if something doesn't work.